src/modules/deposit/model/deposit.entity.ts

```diff
@@ -37,6 +37,23 @@ export type RequestedSpeedUpDepositTx = {
   updatedRecipient?: string;
   updatedMessage?: string;
 };
+export type FeeBreakdown = {
+  // lp fee
+  lpFeeUsd: string;
+  lpFeePct: string; // wei pct
```

Comment:
We may not do it in other places, but should this be a NUMERIC()?

Reply:
As you can see [here](https://github.com/across-protocol/scraper-api/pull/276/files#diff-f9ec4a8d90d4d98e29761006de9633490a8df532a510b886b1c51713a8377c2aR149-R151), we are storing the fee breakdown as a `jsonb` type. I don't think it is possible to specify SQL specific types within `jsonb` 🤔